README.md

```diff
@@ -1,12 +1,60 @@
+
+## Features and Benefits
+
+## Customization
+
+The default appearance of the Blazor Spinner component can be [customized](https://blazor.syncfusion.com/documentation/spinner/customization?utm_source=github&utm_medium=listing&utm_campaign=blazor-spinner-github-samples) by changing its color and stroke width. Also, you can set the z-index for Blazor spinners dynamically based on application scenarios.
+
+## Visibility
+
+With the use of two-way binding and asynchronous methods, you can simply adjust the visibility of the Blazor Spinner component.
+
+## Descriptive labels
+
+Labels that provide extra, valuable details about the task that is presently running should be included with the Blazor Spinner component.
+
+## Related links
+
+[Learn More about Blazor Spinner](https://www.syncfusion.com/blazor-components/blazor-spinner?utm_source=github&utm_medium=listing&utm_campaign=blazor-spinner-github-samples)
+
+[Download Free Trial](https://www.syncfusion.com/downloads/blazor?utm_source=github&utm_medium=listing&utm_campaign=blazor-spinner-github-samples)
+
+[Pricing](https://www.syncfusion.com/sales/products/blazor?utm_source=github&utm_medium=listing&utm_campaign=blazor-spinner-github-samples)
+
+[Documentation](https://blazor.syncfusion.com/documentation/spinner/getting-started?utm_source=github&utm_medium=listing&utm_campaign=blazor-spinner-github-samples)
+
+[View Demos](https://github.com/SyncfusionExamples/getting-started-with-the-blazor-spinner-component?utm_source=github&utm_medium=listing&utm_campaign=blazor-spinner-github-samples)
+
+[Community Forums](https://www.syncfusion.com/forums/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-spinner-github-samples)
+
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-spinner-github-samples)
+
+[Online example](https://blazor.syncfusion.com/demos/spinner/default-functionalities?utm_source=github&utm_medium=listing&utm_campaign=blazor-spinner-github-samples)
+
+[Video Tutorials](https://www.syncfusion.com/tutorial-videos/blazor/spinner?utm_source=github&utm_medium=listing&utm_campaign=blazor-spinner-github-samples)
+
+[Knowledge base](https://www.syncfusion.com/kb/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-spinner-github-samples)
+
+## About Syncfusion Blazor Components
+The Syncfusion's [Blazor components](https://www.syncfusion.com/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-spinner-github-samples) library offers over 70 UI components to work with Blazor server-side and client-side (Blazor WebAssembly) projects seamlessly. In addition to Spinner, we provide popular Blazor Components such as DataGrid,Charts, Scheduler, Diagram and Word Processor.

```

Comment:
need to add FT links for the controls
![image](https://user-images.githubusercontent.com/61148953/176996435-c92350a6-bb9f-4427-ae2a-b43a5acb6b87.png)